test/fixtures/html/integration/_types/lists/index.html

```diff
@@ -60,7 +60,7 @@
     <link href="https://beta.parliament.uk/search/opensearch" rel="search" title="UK Parliament" type="application/opensearchdescription+xml">
     <script src="appInsights.js"></script>
     <script src="googleAnalytics.js"></script>
-    <meta content="Lists - Types - UK Parliament" property="og:title">
+    <meta content="_types.lists.title" property="og:title">
```

Comment:
There should not be translation keys in the HTML. By this time they should have been rendered into the value text.